Dockerfile

```
@@ -263,4 +263,7 @@ COPY setup/phpunit/problem-matcher.json /etc/laminas-ci/problem-matcher/phpunit.
 
 RUN useradd -ms /bin/bash testuser
 
+# Copy ubuntu setup
+COPY setup/ubuntu /
```

This will copy the files with the ownership of the current user. If that needs to be root you might want to 
```suggestion
COPY --chown=root:root setup/ubuntu /
```

---

It seems that it is already `root`:

```
→ docker run -it --entrypoint /bin/bash -v $(realpath .):/github/workspace -w=/github/workspace docker.io/laminas/laminas-ci-container-action:dev-local -c bash
root@b1fd84a4ef52:/github/workspace# ls -l /etc/security/limits.d/testuser.conf 
-rw-r--r-- 1 root root 58 Dec 16 17:40 /etc/security/limits.d/testuser.conf
```